mediaflow_proxy/utils/http_utils.py

```
@@ -1,644 +1,748 @@
+        
+        # Cache the result
+        _dlhd_extraction_cache[destination] = {
+            "data": result,
+            "timestamp": current_time
+        }
+        logger.info(f"DLHD data cached for {_dlhd_cache_duration}s")
+        
+        return result
+        
+    except (ExtractorError, DownloadError) as e:
+        logger.error(f"DLHD extraction failed: {str(e)}")
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)